system/Router/RouteCollection.php

```diff
@@ -328,13 +328,13 @@ public function setAutoRoute(bool $value): RouteCollectionInterface
 	 *
 	 * This setting is passed to the Router class and handled there.
 	 *
-	 * @param callable|null $callable
+	 * @param string|callable|null $override404
 	 *
 	 * @return RouteCollectionInterface
 	 */
-	public function set404Override($callable = null): RouteCollectionInterface
+	public function set404Override($override404 = null): RouteCollectionInterface
```

Comment:
the argument variable change from `$callable` to `$override404` is essential here `$callable` is just like it must be `callable or null`, while it can be string as well